invenio/legacy/bibsched/__init__.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of Invenio.
+# Copyright (C) 2014, 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Legacy BibSched."""
+
+import warnings
+
+from invenio.utils.deprecation import RemovedInInvenio23Warning
+
+warnings.warn("Legacy BibSched will be removed in 2.3.",
```

Comment:
Yes, Celery is ready and we even refer to it *lightly* in the docs already... what I had in mind was mostly indexing/ranking that still needs bibsched.  When these will go away, and when we have new upload mechanism in place, we can advertise new "task scheduling" more prominently, e.g. via before-insert and after-insert hooks...